lodash.js

```diff
@@ -1976,8 +1976,7 @@
             }
           }
           result.push(value);
-        }
-        else if (indexOf(values, value, 0) < 0) {
+        } else if (indexOf(values, value, 0) < 0) {
```

Comment:
I'm not nit picky on the `} else if` bit if it's able to just let that slide (I use whichever is more readable for the situation).